### PR TITLE
add sedai

### PR DESCRIPTION
**Pull request by pwujek**
### What does this PR do?

Creates a Tile for Sedai

### Motivation

Sedai requires an integration with DataDog for marketing purposes.

### Review checklist

- [ ] PR has a [meaningful title](https://github.com/DataDog/integrations-core/blob/master/CONTRIBUTING.md#pull-request-title) or PR has the `no-changelog` label attached
- [ ] Feature or bugfix has tests
- [ ] Git history is clean
- [ ] If PR impacts documentation, docs team has been notified or an issue has been opened on the [documentation repo](https://github.com/DataDog/documentation/issues/new)

### Additional Notes
none

### CODE REVIEW

File: sedai/manifest.json [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    "automation",
+    "aws",
+    "azure",
+    "cloud",
+    "cost-management",
+    "google-cloud",
+    "marketplace",
+    "notification",
+    "orchestration",
+    "provisioning"
```
Comment (coignetp) [inline sedai/manifest.json:30]:
```suggestion
    "automation",
    "cloud",
    "cost-management",
    "notification",
    "orchestration",
    "provisioning"
```

File: sedai/README.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+| Professional Tier 3 | Starter + Remediations |     2,000     |       500M 	    | $4,800  | $48,000   |
+
+### Additional usage fees
+You will be billed monthly for additional usage costs if your usage exceeds your contract. Your additional usage costs will be determined by the number of units you use above your contract.
+
+| Tier       	      | per instance/pod | per 10k invocations |
+| :---       	      | :--------------: | :-----------------: |
+| Professional Tier 1 |      $3.40       |       $0.170        |
+| Professional Tier 2 |      $2.60       |       $0.130        |
+| Professional Tier 3 |      $2.40       |       $0.096        |
```
Comment (coignetp) [inline sedai/README.md:48]: Could this be removed from the readme, and instead have a link to sedai website having the latest information?

**Review by coignetp — CHANGES_REQUESTED** [review]

File: sedai/README.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
+# Autonomous Cloud Application Management
+
+*Autonomous is your future; Production is the new playground*
+
+## **[Schedule a Demo][2]**
```
Comment (coignetp) [inline sedai/README.md:5]:
```suggestion
```
Let's have this section in one place only

**Review by coignetp — CHANGES_REQUESTED** [review]

File: sedai/README.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
+# [Sedai][1] — Find out why production is the new playground with a free 14 day trial
+## Continuous Autonomous Availability Management (CAAM)
+
+*Intelligent enough to detect, independent enough to act — that's the power of autonomous*
+
+## Overview
```
Comment (anawishnoff) [inline sedai/README.md:6]: Under the Overview tab, I'd make it clear that Sedai is offering an integration with Datadog, and that if you're already a customer of Sedai you can set up the integration quickly by following the instructions in the Configuration tab.

File: sedai/README.md (hunk 3 above)
Comment (pwujek) [reply sedai/README.md]: done

File: sedai/README.md (hunk 2 above)
Comment (pwujek) [reply sedai/README.md]: done

File: sedai/manifest.json (hunk 1 above)
Comment (pwujek) [reply sedai/manifest.json]: done

**Review by anawishnoff — CHANGES_REQUESTED** [review]

File: sedai/README.md [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+* Increase team productivity and agility by 3x
+
+## Core Features
+* **Easy setup:** Our agentless approach integrates with your cloud accounts and observability platforms to get up and running in minutes.
+* **Configuration-free:** We infer your topology and metrics, and intelligently leverage them to detect signals based on observed performance.
+* **Autonomous Availability:** Prevent issues and ensure the highest levels of availability with autonomous actions.
+* **Release Intelligence:** Validate releases' performance in production in real-time with smart scorecards that assess the quality of your deployments and analyze deviations in latency and errors.
+* **Resource Optimization:** Improve customer experience and lower cloud costs on the fly with dynamic allocation that smartly manages your resources based on their traffic needs.
+
+## Home Page
```
[the remark this answers is not in the document]
Comment (pwujek) [reply sedai/README.md:22]: changed to "Visit **[Sedai Inc.][1]** to learn more.**"

File: sedai/README.md (hunk 4 above)
Comment (pwujek) [reply sedai/README.md:6]: done

File: sedai/README.md [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+### Ready to Autonomously Optimize your applications for heightened performance?
+With our agentless approach, integration is seamless and setup takes minutes — simply connect your cloud and monitoring accounts and favorite integrations. We automatically identify your topology and metrics and start analyzing performance immediately.
+
+**[Schedule a free 30 minute demo to learn more][2]**
+
+**[Schedule a free 14-day trial][2]**
+
+[Contact us][5] if you have questions about the process.
+
+For existing Sedai customers, **[learn how to receive notifications in DataDog about Sedai activity.][6]**
```
[the remark this answers is not in the document]
Comment (pwujek) [reply sedai/README.md:35]: Sedai customers who are also DataDog customers will already have an integration set up on Sedai's side, the instructions tell them how to configure specific notifications about Sedai activity to appear on their DataDog Dashboard.

File: sedai/manifest.json [hunk 7]
```diff
@@ -0,0 +1,38 @@
+{
+  "display_name": "Sedai",
+  "maintainer": "Paul.Wujek@sedai.io",
+  "manifest_version": "1.0.0",
+  "name": "sedai",
+  "metric_prefix": "sedai.",
+  "metric_to_check": "",
+  "creates_events": false,
+  "short_description": "an autonomous SaaS that intelligently manages your cloud environments",
```
[the remark this answers is not in the document]
Comment (pwujek) [reply sedai/manifest.json:9]: done

File: sedai/README.md [hunk 8]
```diff
@@ -0,0 +1,47 @@
+# [Sedai][1] — Find out why production is the new playground with a free 14 day trial
+## Continuous Autonomous Availability Management (CAAM)
```
[the remark this answers is not in the document]
Comment (pwujek) [reply sedai/README.md:2]: done

Comment (anawishnoff) [inline sedai/README.md:2]: We'll have to remove this level 2 heading as well

File: sedai/README.md [hunk 9]
```diff
@@ -0,0 +1,47 @@
+# [Sedai][1] — Find out why production is the new playground with a free 14 day trial
```
Comment (anawishnoff) [inline sedai/README.md:1]: I don't believe level 1 headings are supported either - I'd convert this to a level 3 or just leave it as body text.

File: sedai/README.md [hunk 10]
```diff
@@ -0,0 +1,47 @@
...
+* Increase team productivity and agility by 3x
+
+### Core Features
+* **Easy setup:** Our agentless approach integrates with your cloud accounts and observability platforms to get up and running in minutes.
+* **Configuration-free:** We infer your topology and metrics, and intelligently leverage them to detect signals based on observed performance.
+* **Autonomous Availability:** Prevent issues and ensure the highest levels of availability with autonomous actions.
+* **Release Intelligence:** Validate releases' performance in production in real-time with smart scorecards that assess the quality of your deployments and analyze deviations in latency and errors.
+* **Resource Optimization:** Improve customer experience and lower cloud costs on the fly with dynamic allocation that smartly manages your resources based on their traffic needs.
+
+Visit **[Sedai Inc.][1]** to learn more.**
```
Comment (anawishnoff) [inline sedai/README.md:25]:
```suggestion
Visit **[Sedai Inc.][1]** to learn more.
```

File: sedai/README.md [hunk 11]
```diff
@@ -0,0 +1,47 @@
...
+
+### Technical Documentation
+Please review the [Sedai User Guide][4].
+## Setup
+Existing Sedai customers can **[learn how to receive notifications in DataDog about Sedai activity.][6]**
```
Comment (anawishnoff) [inline sedai/README.md:30]:
```suggestion
Existing Sedai customers can receive notifications in DataDog about Sedai activity. Learn how to connect Datadog to Sedai in **[our documentation][6]**
```

File: sedai/README.md (hunk 8 above)
Comment (pwujek) [reply sedai/README.md]: done

File: sedai/README.md (hunk 9 above)
Comment (pwujek) [reply sedai/README.md]: done

File: sedai/README.md (hunk 10 above)
Comment (pwujek) [reply sedai/README.md]: done

File: sedai/README.md (hunk 11 above)
Comment (pwujek) [reply sedai/README.md]: done

File: sedai/README.md [hunk 12]
```diff
@@ -0,0 +1,47 @@
...
+
+## Support
+Support is provided 24/7 via email to [support@sedai.io][3].
+
+[1]:[https://www.sedai.io]
+[2]:[https://calendly.com/sedai/sedai-demo]
+[3]:[email:support@sedai.io]
+[4]:[https://sedai.gitbook.io/sedai/]
+[5]:[email:contact@sedai.io]
+[6]:[https://sedai.gitbook.io/sedai/sedai-user-guide/controls/notifications]
```
Comment (alai97) [inline sedai/README.md:47]:
```suggestion
[1]: https://www.sedai.io
[2]: https://calendly.com/sedai/sedai-demo
[3]: mailto:support@sedai.io
[4]: https://sedai.gitbook.io/sedai/
[5]: mailto:contact@sedai.io
[6]: https://sedai.gitbook.io/sedai/sedai-user-guide/controls/notifications
```

File: sedai/README.md [hunk 13]
```diff
@@ -0,0 +1,47 @@
...
+* **Easy setup:** Our agentless approach integrates with your cloud accounts and observability platforms to get up and running in minutes.
+* **Configuration-free:** We infer your topology and metrics, and intelligently leverage them to detect signals based on observed performance.
+* **Autonomous Availability:** Prevent issues and ensure the highest levels of availability with autonomous actions.
+* **Release Intelligence:** Validate releases' performance in production in real-time with smart scorecards that assess the quality of your deployments and analyze deviations in latency and errors.
+* **Resource Optimization:** Improve customer experience and lower cloud costs on the fly with dynamic allocation that smartly manages your resources based on their traffic needs.
+
+Visit **[Sedai Inc.][1]** to learn more.
+
+### Technical Documentation
+Please review the [Sedai User Guide][4].
```
Comment (alai97) [inline sedai/README.md:28]: We include a link to the user guide and the documentation already below.

```suggestion
```

File: sedai/README.md [hunk 14]
```diff
@@ -0,0 +1,47 @@
...
+## Setup
+Existing Sedai customers can receive notifications in DataDog about Sedai activity. Learn how to connect DataDog to Sedai in **[our documentation][6].**
```
Comment (alai97) [inline sedai/README.md:30]: [6] links to https://sedai.gitbook.io/sedai/sedai-user-guide/controls/notifications, not https://sedai.gitbook.io/sedai/sedai-user-guide/controls/integrations/notifications-integrations. There are two sections titled "Notifications" so I want to confirm we're linking to the right one!  

```suggestion

After connecting your cloud provider, monitoring provider, and notification provider accounts, Sendai identifies your topology and metrics and starts to analyze performance.

### Installation

Existing Sedai customers can receive notifications about Sedai activity in Datadog. For more information, see the [Sedai documentation][6].
```

File: sedai/README.md [hunk 15]
```diff
@@ -0,0 +1,47 @@
...
+
+### Ready to Autonomously Optimize your applications for heightened performance?
+With our agentless approach, integration is seamless and setup takes minutes — simply connect your cloud and monitoring accounts and favorite integrations. We automatically identify your topology and metrics and start analyzing performance immediately.
```
Comment (alai97) [inline sedai/README.md:33]:
```suggestion
```

File: sedai/README.md [hunk 16]
```diff
@@ -0,0 +1,47 @@
...
+
+**[Schedule a free 30 minute demo OR a free 14-day trial][2]**
```
Comment (alai97) [inline sedai/README.md:35]:
```suggestion
## Data Collected

### Metrics

This integration does not collect metrics.

### Events

This integration does not support events.

### Service

This integration does not collect service checks.

## Support

Need help? Email [Support][3] is available 24/7.

Alternatively, [schedule a 30-minute demo][2] or [contact us][5] for a [free 14-day trial][2]. 
```

File: sedai/README.md [hunk 17]
```diff
@@ -0,0 +1,47 @@
...
+
+[Contact us][5] if you have questions about the process.
+
+## Support
+Support is provided 24/7 via email to [support@sedai.io][3].
```
Comment (alai97) [inline sedai/README.md:40]:
```suggestion
```

**Review by alai97 — CHANGES_REQUESTED** [review]
Doc Review!

File: sedai/README.md (hunk 14 above)
Comment (pwujek) [reply sedai/README.md:26]: The two links are on purpose, our UI has 2 related settings pages that are involved in configuration of notifications.

File: sedai/README.md [hunk 18]
```diff
@@ -0,0 +1,61 @@
...
+### Core Features
+
+* **Easy setup:** Sedai's agentless approach integrates with your cloud accounts and observability platforms so you can get up and running in minutes.
+* **Configuration-free:** By inferring your topology and metrics, Sedai leverages them to detect signals based on observed performance.
+* **Autonomous Availability:** Prevent issues and ensure the highest levels of availability with autonomous actions.
+* **Release Intelligence:** Validate release performance in production in real-time with smart scorecards that assess the quality of your deployments and analyze deviations in latency and errors.
+* **Resource Optimization:** Improve customer experience and lower cloud costs on the fly with dynamic allocation that manages your resources based on traffic needs.
+* **Smart SLOs:** In addition to automatically analyzing and defining targets, Sedai manages error budgets with autonomous actions that prevent errors, downtime, and SLO breaches.
+
+To learn more, see [Sedai Inc.][1].
```
Comment (anawishnoff) [inline sedai/README.md:24]:
```suggestion
To learn more, see [Sedai Inc.][1]
```

File: sedai/metadata.csv [hunk 19]
```diff
@@ -0,0 +1 @@
+metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name
```
Comment (anawishnoff) [inline sedai/metadata.csv:1]: If there are no metrics to list here, I believe this file can be deleted. @coignetp could you confirm?

**Review by alai97 — APPROVED** [review]
Looks great, thanks @pwujek! 🥳

**Review by coignetp — CHANGES_REQUESTED** [review]
Almost all good, just need the metadata.csv file to be deleted

File: sedai/README.md [hunk 20]
```diff
@@ -0,0 +1,61 @@
...
+* **Autonomous Availability:** Prevent issues and ensure the highest levels of availability with autonomous actions.
+* **Release Intelligence:** Validate release performance in production in real-time with smart scorecards that assess the quality of your deployments and analyze deviations in latency and errors.
+* **Resource Optimization:** Improve customer experience and lower cloud costs on the fly with dynamic allocation that manages your resources based on traffic needs.
+* **Smart SLOs:** In addition to automatically analyzing and defining targets, Sedai manages error budgets with autonomous actions that prevent errors, downtime, and SLO breaches.
+
+To learn more, see [Sedai Inc.][1]
+
+## Setup
+
+After connecting your cloud provider, monitoring provider, and notification provider accounts, Sendai identifies your topology and metrics and starts to analyze performance.
```
Comment (anawishnoff) [inline sedai/README.md:28]: Doing a final read-through and found this typo: 
```suggestion
After connecting your cloud provider, monitoring provider, and notification provider accounts, Sedai identifies your topology and metrics and starts to analyze performance.
```

File: sedai/metadata.csv (hunk 19 above)
Comment (pwujek) [reply sedai/metadata.csv]: deleted

**Review by coignetp — APPROVED** [review]

**Review by anawishnoff — APPROVED** [review]

**Pull request merged**